### PR TITLE
[APB-3312][MP] Remove NiOrg code, revert select-service-business page

### PR DESCRIPTION
**Pull request by mariannepearson0**

### CODE REVIEW

File: app/uk/gov/hmrc/agentinvitationsfrontend/controllers/BaseController.scala [hunk 1]
```diff
@@ -50,30 +50,26 @@ abstract class BaseController(
   val vat =
     if (featureFlags.showHmrcMtdVat) Seq(HMRCMTDVAT -> Messages("select-service.vat")) else Seq.empty
 
-  val niOrg =
```
Comment (sureshhmrc) [inline app/uk/gov/hmrc/agentinvitationsfrontend/controllers/BaseController.scala]: i guess we can remove the `BaseController.scala` completely as it only contains a few messages and move its logic to `BaseInvitationController`

File: app/uk/gov/hmrc/agentinvitationsfrontend/controllers/BaseController.scala [hunk 2]
```diff
@@ -50,30 +50,26 @@ abstract class BaseController(
...
-
   def enabledPersonalServicesForCancelAuth(isWhitelisted: Boolean): Seq[(String, String)] =
     if (isWhitelisted) {
       personalIncomeRecord ++ mtdItId ++ vat
     } else {
       mtdItId ++ vat
     }
 
-  def enabledBusinessServicesForCancelAuthorisation: Seq[(String, String)] = vat ++ niOrg
+  def enabledBusinessServicesForCancelAuthorisation: Seq[(String, String)] = vat
```
Comment (sureshhmrc) [inline app/uk/gov/hmrc/agentinvitationsfrontend/controllers/BaseController.scala]: i think removing this def makes much clear, just use `vat` where ever we call this

Comment (mariannepearson0) [reply app/uk/gov/hmrc/agentinvitationsfrontend/controllers/BaseController.scala]: 👍

Comment (mariannepearson0) [reply app/uk/gov/hmrc/agentinvitationsfrontend/controllers/BaseController.scala]: It's actually not used anywhere haha

File: app/uk/gov/hmrc/agentinvitationsfrontend/views/agents/business_select_service.scala.html [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+@uk.gov.hmrc.agentinvitationsfrontend.views.html.main_template(title = error_prefix(choiceForm) + Messages("generic.title", Messages("business-select-service.header"), Messages("title.suffix.agents")), bodyClasses = None, isAgent = true) {
+
+    @back_link_referer(request)
+
+    @if(choiceForm.hasErrors) {
+        @errorSummary(Messages("error.summary.heading"), choiceForm, Seq.empty, Some("service"))
+    }
+
+    @form(
+        action = routes.AgentsInvitationController.submitSelectService(),
```
Comment (sureshhmrc) [inline app/uk/gov/hmrc/agentinvitationsfrontend/views/agents/business_select_service.scala.html]: this still wont work  if we are in cancellation-journey :)

**Pull request merged**